ZelBack/src/services/fluxCommunication.js

```diff
@@ -93,6 +93,28 @@ async function getMyFluxIP() {
   return myIP;
 }
 
+async function getBenchPublicIp() {
```

Comment:
this function getBenchPublicIp and restartNodeBenchMarks are actually not needed, can be removed and simplified.
Please paste similar code directly to checkMyFluxAvailability where you are calling the 
await daemonService.restartNodeBenchmarks(); directly

Reply:
Sorry, this part didn't understood